docs/wiki/development/building.md

```diff
@@ -268,6 +268,8 @@ OSQUERY_BUILD_DEPS=True # Install dependencies from source when using make deps
 OSQUERY_BUILD_BOTTLES=True # Create bottles from installed dependencies
 OSQUERY_BUILD_VERSION=9.9.9 # Set a wacky version string
 OSQUERY_PLATFORM=custom_linux;1.0 # Set a wacky platform/distro name
+OSQUERY_OSQUERY_DEPS=/usr/local/osquery # Set alternative dependency path
```

Comment:
Looks like this might be a bad merge?

Reply:
@muffins  Sorry, the merge was an honest mistake. Maybe I should delete my dev branch and close this PR. Then create another with a fresh branch. Is this okay?